ydb/tests/sql/large fails in test_insert_delete_duplicate_records

[fail] test_insert_delete_duplicate_records.py::TestConcurrentInsertDeleteAndRead::test_bulkupsert_delete_and_read_tpch [default-linux-x86_64-profile] (33.16s)
E   AssertionError: Unexpected count of rows with l_suppkey=9999: 564
E   assert 564 in {0, 1}

...
[fail] test_insert_delete_duplicate_records.py::TestConcurrentInsertDeleteAndRead::test_upsert_delete_and_read_tpch [default-linux-x86_64-profile] (39.46s)
E   AssertionError: Unexpected count of rows with l_suppkey=9999: 564
E   assert 564 in {0, 1}

[fail] test_insert_delete_duplicate_records.py::TestConcurrentInsertDeleteAndRead::test_upsert_delete_and_read_tpch_tx [default-linux-x86_64-profile] (41.17s)
E   AssertionError: Unexpected count of rows with l_suppkey=9999: 563
E   assert 563 == 0

--- a/ydb/tests/sql/large/test_insert_delete_duplicate_records.py
+++ b/ydb/tests/sql/large/test_insert_delete_duplicate_records.py
@@ -183,6 +183,12 @@ class TestConcurrentInsertDeleteAndRead(TpchTestBaseH1):
             while not all(f.done() for f in insert_futures):
                 check_data()
 
+        # Delete all records with l_suppkey = 9999
+        delete_query = f"""
+        DELETE FROM `{lineitem_table}` WHERE l_suppkey = 9999;
+        """
+        self.query(delete_query)
+
         # Use ThreadPoolExecutor to manage threads
         with concurrent.futures.ThreadPoolExecutor(max_workers=num_threads+1) as executor:
             # Start insert and delete operations

